### DISCUSSION TITLE
Running on a Steam Deck?

### DISCUSSION THREAD

**Discussion by dafunks** [Luois45/claim-free-steam-packages#106] [Ideas]
Just got myself a Steam Deck and I was thinking that it was be kinda funny but also very handy having this run on a Steam Deck. Do you think its possible?

**Comment by louisa-uno** [comment]
I think that it would work, but I'm not sure about it.
It could help to take a look into this: https://github.com/mikeroyal/Steam-Deck-Guide#getting-software

**Reply by dafunks** [reply]
I'll check it out. I think it'll take a week or so to get here. I will let you know if i get it working.
